app/controllers/contributors_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ContributorsController < ApplicationController
+  before_action :check_if_feature_flag_is_on
+
+  def new
+    @contributor = Contributor.new
+  end
+
+  def create
+    @contributor = Contributor.new(contributor_params)
+    @suggested_councillor = SuggestedCouncillor.find(params[:contributor][:suggested_councillor_id])
+    if @contributor.save
+      @suggested_councillor.update(contributor: @contributor)
```

Comment:
If this update doesn't happen for some reason then no one will know. It's better to use the bang method here, `#update!`, to fail fast and hard.